lib/providers/gcm.js

```diff
@@ -8,6 +8,8 @@ var extend = require('util')._extend;
 var EventEmitter = require('events').EventEmitter;
 var gcm = require('node-gcm');
 var debug = require('debug')('loopback:component:push:provider:gcm');
+var SG = require('strong-globalize');
```

Comment:
Why not following strong-globalize coding guideline here?
https://www.npmjs.com/package/strong-globalize#autonomous-message-loading
Please study paragraphs after "For example, the following does not work as intended because the package sub calls SG.SetRootDir first:"